x-pack/platform/test/api_integration_deployment_agnostic/apis/streams/attachments/attachments.ts

```diff
@@ -0,0 +1,398 @@
+/*
```

Comment:
Can you add some tests on what happens when you try to link a dashboard that does not exist?

Reply:
I've added more tests for cross-space linking. When a user tries to link a dashboard that doesn't exist or a dashboard from another space, both cases return 404. This is because the saved object client we use to fetch dashboards is space-aware and treats dashboards in other spaces the same as non-existent dashboards.